### ISSUE TITLE
[FEATURE] Proton Prefix

### ISSUE THREAD

**Issue by massatt212** [Vysp3r/ProtonPlus#142]
the ability to delete proton prefixes on desktop mode, without hunting down the folder location

**Comment by Vysp3r** [comment]
I don't know where those are, do you have an example with more data for me?

**Comment by massatt212** [comment]
Steam compatdata folder

**Comment by Vysp3r** [comment]
That would delete every proton prefixes tho no? You would lose a lot of data if not backed up properly. I would need to add a way to list the games you have installed to delete a specific prefix and I don't plan on doing the game list part of ProtonPlus right now.

**Comment by Vysp3r** [comment]
Or maybe I don't understand completely the prefixes, if so just try to explain it to me in a simple way so that I can understand what you really want.

**Comment by massatt212** [comment]
Alright np, but deleting data is not a problem just got to use a warning message before you do, and take your time with it.

**Comment by massatt212** [comment]
The proton or wine prefix install the game files like directx vcrun and so on for proton it get corrupted when changing proton versions sometimes, so they tend to say delete it and install fresh, cause steam deck has one built in to delete it, but desktop users have to find the id and delete it manually, a program toml speed that up would  be very nice.

**Comment by massatt212** [comment]
Yes delete specific game, and game cloudless games, that will be there responsibility,that's why u can give them a detail warning ⚠️

**Comment by Vysp3r** [comment]
I will add that feature for the next update.

**Comment by Vysp3r** [comment]
I just thought about, but if you want to be able to delete the proton prefixes for specific games that will not be possible for now. Not until I implement a way to list Steam games and so on. Which is why I will put this on standby for now.

**Comment by massatt212** [comment]
> I just thought about, but if you want to be able to delete the proton prefixes for specific games that will not be possible for now. Not until I implement a way to list Steam games and so on. Which is why I will put this on standby for now.

Take a look at proton tricks, they can tell which exact steam game it is.

**Comment by Vysp3r** [comment]
Most of them are you using python which makes it easy for them. I have to do most thing by myself hence why my application is way smaller in size then ProtonUp-Qt. I planned on adding the 'Games' tab for v0.5.

**Comment by massatt212** [comment]
Ok

**Comment by Arcitec** [comment]
We have discussed it while triaging old issues, and decided that it's out of scope.

- If you uninstall a game in Steam, it automatically deletes the `compatdata` for that game.
- If you are still using a game, you need its `compatdata`. People don't go around deleting it themselves.
- And if you really want to delete a `compatdata` yourself, use https://steamdb.info/ to find the ID of the folder to delete.

**Comment by Vysp3r** [comment]
It came to my knowledge that the compatdata directories are deleted automatically once you uninstall the game so I will close this request since I don't really see a need for my application to do so.

If you feel like I've missed something just tell me and I will re-open the issue.
